src/main/java/uk/gov/register/util/DateFormat.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.register.util;
+
+public interface DateFormat {
```

Comment:
I'd be interested to hear what others think, but to me this feels like we should create these variables as `public static` inside of a class, as opposed to an interface.

Reply:
More opinions about this?

Reply:
Erm, I think I agree with @karlbaker02 that since there is nothing directly implementing this interface, that it makes more sense as`public static`?

Reply:
I was just think about a centrilized place to have all the date formats not just a class where to burry them. But, if both of you agree to change it, I'll do it

Reply:
I don't mind this being in it's own interface but I don't have strong feelings about either solution. Happy with either.

Reply:
I prefer to have it defined on it's own considering is a format we use in the system